v3.9/getting-started/kubernetes/hardway/install-typha.md

```diff
@@ -129,11 +129,11 @@ Bind the cluster role to the `calico-typha` ServiceAccount.
 kubectl create clusterrolebinding calico-typha --clusterrole=calico-typha --serviceaccount=kube-system:calico-typha
 ```
 
-## Install Deployment
+## Install Typha Deployment
 
-Since Typha is required by `calico/node`, and `calico/node` establishes the pod network, we run Typha as a host networked pod to avoid
-a chicken-and-egg problem.  We run 3 replicas of Typha so that even during a rolling update, a single failure does not
-make Typha unavailable.
+The pod network is *not* available when initially installing `calico/node`, and if using `calico/node` with `typha`, typha is required for pod network establishment. Thus, we cannot rely on the pod network at the time of `calico/node` installation in this scenario, and we bind `typha` to the `hostNetwork`, for direct, non k8s communication during bootstrapping.  We run 3 replicas of Typha so that even during a rolling update, a single failure does not make Typha unavailable.  
```

Comment:
Probably best to standardise on "Typha" everywhere (You've got typha, `typha` and Typha all in this one paragraph.)